src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -37,7 +37,7 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     if (!layer.name) {
         throw new Error('layerName is required.');
     }
-    if (!layer.bbox) {
+    if (!layer.options && !layer.options.extent) {
```

Comment:
Also please fix the error message below: 'layer.options.extent property is required'.
Btw, it's a bit strange to move `extent` in `options` since it's a mandatory parameter.

IMHO, the `options` thing should go away, and all layer properties should live at the top level. What do you think?